### PR TITLE
`Development`: Match server and client run configurations

### PR DESCRIPTION
**Pull request by 4ludwig4**
#### General
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

I guess webpack is just used for the dev profile and is baked into the gradle build as for me this configuration failed by default?!

### Description

Building the webapp via gradle instead of running the deleted webpack gradle task and a weird make for the server.
The webpack gradle task was deleted [in this PR](https://github.com/ls1intum/Artemis/pull/4022).

### Steps for Testing
Try if it builds what you expect.
Do you expect a different build for developing on the server or client code?

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### CODE REVIEW

File: .idea/runConfigurations/Artemis__Server___Client_.xml [hunk 1]
```diff
@@ -8,8 +8,7 @@
     <option name="ALTERNATIVE_JRE_PATH" />
     <option name="SHORTEN_COMMAND_LINE" value="MANIFEST" />
     <method v="2">
-      <option name="Gradle.BeforeRunTask" enabled="true" tasks="webpack" externalProjectPath="$PROJECT_DIR$" vmOptions="" scriptParameters="" />
-      <option name="Make" enabled="true" />
+      <option name="Gradle.BeforeRunTask" enabled="false" tasks="build" externalProjectPath="$PROJECT_DIR$" vmOptions="" scriptParameters="-x test -x jacocoTestCoverageVerification -x spotlessCheck -x checkstyleMain -x checkstyleTest" />
```
Comment (sleiss) [inline .idea/runConfigurations/Artemis__Server___Client_.xml:11]: Is it intended that `enabled` is set to false now?

Comment (4ludwig4) [reply .idea/runConfigurations/Artemis__Server___Client_.xml:11]: Couldn't find the definition of the runConfigurations XML. I wasn't able to change it from false to true by using the UI. Also the other runConfigs use `enabled="false"` so I guess it should be fine.

**Review by sleiss — APPROVED** [review]
Tested it locally, LGTM.

**Review by frederik-kt — APPROVED** [review]
Ran the Artemis (Server & Client) profile locally. Looks good!

**Pull request merged**